tequilapi/validation/validation.go

```diff
@@ -38,6 +38,11 @@ func (fel *FieldErrorList) AddError(code string, message string) {
 	fel.list = append(fel.list, FieldError{code, message})
 }
 
+// HasErrors return true if at least one field errors exist
+func (fel *FieldErrorList) HasErrors() bool {
+	return fel.list != nil && len(fel.list) > 0
```

Comment:
I would argue that `fel.list != nil` check is superfluous - https://play.golang.org/p/HbtAjHBtTJ8